app/src/main/java/app/foodfinderapp/dto/Food.kt

```diff
@@ -1,6 +1,11 @@
 package app.foodfinderapp.dto
 
-data class Food(var foodID:Int, var foodName: String, var foodPrice: Float, var foodCategory: String) {
+data class Food(
+    val foodID:Int,
+    var foodName: String,
+    var foodPrice: Float,
+    var foodCategory: String
+    ) {
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.